app/views/endemics/guidance/apply-guidance-for-farmers.njk

```diff
@@ -212,7 +212,7 @@
               <p>Rural Payments Agency<br>PO Box 352<br>Worksop<br>S80 9FG<br></p>
               <p>Email: <a href="mailto:ruralpayments@defra.gov.uk" class="govuk-link">{{ruralPaymentsAgency.email}}</a><br></p>
                <p>Telephone: {{ruralPaymentsAgency.telephone}}<br> Monday to Friday, 8:30am to 5pm, except public holidays<br>
-                <a href="https://www.gov.uk/call-charges" target='_blank' class="govuk-link">Find out about call charges (opens in a new tab)</a>
```

Comment:
This seems right to me. Target should be blank

Reply:
> This seems right to me. Target should be blank

done